loop_ivr/management/commands/product_team_mailer.py

```diff
@@ -0,0 +1,229 @@
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_incoming_sms = get_active_user_comp_info(today_caller_object_sms_count, yesterday_caller_object_sms_count, '')
+
+        # SMS user count
+        today_caller_object_sms_user_count = today_caller_object.filter(call_source=3).values_list('from_number', flat=True).distinct().count()
+        yesterday_caller_object_sms_user_count = yesterday_caller_object.filter(call_source=3).values_list('from_number', flat=True).distinct().count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_sms_users = get_active_user_comp_info(today_caller_object_sms_user_count, yesterday_caller_object_sms_user_count, '')
+
+        print 'SMS users %s'%no_sms_users
+
+        # Correct Queries
+        today_caller_object_correct_query_sms_count = today_caller_object.filter(call_source=3, info_status=1).count()
+        yesterday_caller_object_correct_query_sms_count = yesterday_caller_object.filter(call_source=3, info_status=1).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        per_correct_code_entered_sms = get_active_user_comp_info(today_caller_object_correct_query_sms_count, yesterday_caller_object_correct_query_sms_count, '')
+
+        # Incoming Calls
+        today_caller_object_call_count = today_caller_object.filter(call_source__in=[1, 2]).count()
+        yesterday_caller_object_call_count = yesterday_caller_object.filter(call_source__in=[1, 2]).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_incoming_call = get_active_user_comp_info(today_caller_object_call_count, yesterday_caller_object_call_count, '')
+
+        today_caller_object_correct_query_call_count = today_caller_object.filter(call_source__in=[1, 2], info_status=1).count()
+        yesterday_caller_object_correct_query_call_count = yesterday_caller_object.filter(call_source__in=[1, 2], info_status=1).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        per_correct_code_entered_call = get_active_user_comp_info(today_caller_object_correct_query_call_count, yesterday_caller_object_correct_query_call_count, '')
+
+        # Wrong Code Entered via call
+        today_caller_object_wrong_query_count = today_caller_object.filter(info_status=2).count()
+        yesterday_caller_object_wrong_query_count = yesterday_caller_object.filter(info_status=2).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_wrong_query_code = get_active_user_comp_info(today_caller_object_wrong_query_count, yesterday_caller_object_wrong_query_count, '')
+
+        # Total SMS Sent using textlocal
+        today_smsstatus_obj = SmsStatus.objects.filter(price_info_incoming=today_caller_object)
```

Comment:
today_caller_object is a list. So shouldn't it be price_info_incoming__in

Reply:
today_caller_object is an object that's why it's like this way.